css/style.css

```diff
@@ -24,6 +24,27 @@ a.btn:hover
     background-color: lightgray;
 }
 
+/* replaces <big> tag */
+em.big
+{
+    font-style: normal;
+    font-size: larger;
+}
+
+/* replaces <tt> tag */
+em.tt
+{
+    font-style: normal;
+    font-family: monospace;
+}
+
+/* replaces <u> tag */
+em.u
```

Comment:
I don't think we need to replace `<u>`, at least according to http://www.w3schools.com/tags/tag_u.asp